package.json

```diff
@@ -19,5 +19,10 @@
     "snazzy": "^3.0.0",
     "standard": "^6.0.7",
     "surge": "^0.17.7"
+  },
+  "standard": {
+    "ignore": [
```

Comment:
@fdaciuk, valeu pela dica, coloquei o google como global.
No caso do initMap, mesmo colocando como global eu ainda obtinha o erro:
'initMap' is defined but never used, o mesmo também acontecia para marker.
para esses dois casos, eu coloquei um comentário na linha, para os mesmos serem ignorados, não sei se era a melhor solução, se tiver alguma sugestão eu altero aqui :)